test/integration/start_stop_delete_test.go

```diff
@@ -55,6 +55,9 @@ func TestStartStop(t *testing.T) {
 			"--extra-config",
 			"kubeadm.ignore-preflight-errors=SystemVerification",
 		}},
+		{"apiserver_port_non_default", []string{
+			"--apiserver-port=8444",
```

Comment:
Do you mind combining this with another test entry - such as "containerd"? 

This is less than ideal, but each entry adds about 10 minutes to our integration tests.

Reply:
No problem. I've now pushed a new commit that combines the two tests.